app/modules/import-export-customization/assets/js/import/pages/import-complete.js

```diff
@@ -0,0 +1,159 @@
+import { Box, Typography, Card, CardContent, Button, Link, Stack } from '@elementor/ui';
+import { __ } from '@wordpress/i18n';
+import PropTypes from 'prop-types';
+import { BaseLayout, TopBar, PageHeader, Footer } from '../../shared/components';
+
+const Illustration = () => (
+	<Box
+		sx={ {
+			height: 140,
+			alignSelf: 'center',
+			borderRadius: 2,
+			display: 'flex',
+			alignItems: 'center',
+			justifyContent: 'center',
+		} }
+	>
+		<img
+			src={ elementorAppConfig.assets_url + 'images/kit-is-live.svg' }
+			alt="Kit is live illustration"
+		/>
+	</Box>
+);
+
+const ExternalLinkIcon = ( { fontSize = 20 } ) => (
```

Comment:
You are not customizing it and you shouldn't customizing it :)
And use elementor/icons
```suggestion
const ExternalLinkIcon = () => (
```